[2025-10-08T11:36:03.520+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-10-08T11:36:03.568+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag1.test_pwd scheduled__2025-10-08T11:35:00+00:00 [queued]>
[2025-10-08T11:36:03.582+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag1.test_pwd scheduled__2025-10-08T11:35:00+00:00 [queued]>
[2025-10-08T11:36:03.583+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-10-08T11:36:03.615+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): test_pwd> on 2025-10-08 11:35:00+00:00
[2025-10-08T11:36:03.631+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=201) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-10-08T11:36:03.634+0000] {standard_task_runner.py:63} INFO - Started process 204 to run task
[2025-10-08T11:36:03.635+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dag1', 'test_pwd', 'scheduled__2025-10-08T11:35:00+00:00', '--job-id', '40', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmpfeuhzg7l']
[2025-10-08T11:36:03.639+0000] {standard_task_runner.py:91} INFO - Job 40: Subtask test_pwd
[2025-10-08T11:36:03.665+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:195: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2025-10-08T11:36:03.761+0000] {task_command.py:426} INFO - Running <TaskInstance: dag1.test_pwd scheduled__2025-10-08T11:35:00+00:00 [running]> on host 567e6f4df55c
[2025-10-08T11:36:03.978+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag1' AIRFLOW_CTX_TASK_ID='test_pwd' AIRFLOW_CTX_EXECUTION_DATE='2025-10-08T11:35:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-10-08T11:35:00+00:00'
[2025-10-08T11:36:03.982+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-10-08T11:36:03.984+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-10-08T11:36:03.986+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'pwd']
[2025-10-08T11:36:04.026+0000] {subprocess.py:86} INFO - Output:
[2025-10-08T11:36:04.032+0000] {subprocess.py:93} INFO - /tmp/***tmpbpc37gzb
[2025-10-08T11:36:04.034+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-10-08T11:36:04.035+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-10-08T11:36:04.102+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=dag1, task_id=test_pwd, run_id=scheduled__2025-10-08T11:35:00+00:00, execution_date=20251008T113500, start_date=20251008T113603, end_date=20251008T113604
[2025-10-08T11:36:04.171+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-10-08T11:36:04.217+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-10-08T11:36:04.219+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
